containers/board/SelectBox/index.tsx

```diff
@@ -0,0 +1,76 @@
+import styles from './index.module.scss';
+import Image from 'next/image';
+import { IconArrowDown, IconSort } from '@/public';
+import { MouseEvent, useState } from 'react';
+import Button from '@/components/Button';
+import {
+  SelectdValue,
+  selectList as defaultSelectList,
+} from '@/types/select-order.d';
+
+interface Props {
+  handleSelect: (selectedValue: SelectdValue) => void;
+  selectList: typeof defaultSelectList;
+}
+
+function SelectBox({ handleSelect, selectList = defaultSelectList }: Props) {
+  const [isListOpen, setIsListOpen] = useState(false);
+  const [selectedText, setSelectedText] = useState<SelectdValue>(selectList[0]);
+
+  const handleOpenList = () => {
+    if (isListOpen) {
+      setIsListOpen(() => false);
+    } else {
+      setIsListOpen(() => true);
+    }
```

Comment:
prev 값을 활용하면 더 간단히 작성할 수 있습니다. 😁
```suggestion
 setIsListOpen((prev) => !prev);
```